content/en/service_owners_guide/overview.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Service Owners Guide
+description: Setup and implement Datadog as a Service owner to avoid pit-falls down the road
+further_reading:
+- link: "/getting_started/support/"
+  tag: "Documentation"
+  text: "Getting Started with Datadog Support"
+---
+
+## Overview
+
+When planning any new software implementation, it's crucial to understand its capabilities, objectives, timelines, teams, and design pattern. Throughout the plan phase you will learn some Datadog basics, define your most important objectives, understand several best practices, and identify how to optimize your Datadog implementation. 
```

Comment:
This overview feels out of place - it doesn't accurately describe the contents of this page. This page is kind of a getting started, really.

Reply:
I'll work on this, agree it should be "Getting Started"